lib/serviceBusMessage.ts

```diff
@@ -936,14 +936,14 @@ export class ServiceBusMessage implements ReceivedMessage {
    * rejecting the message.
    * @returns Promise<void>
    */
-  async deadLetter(options?: DeadLetterOptions): Promise<void> {
+  async deadLetter(
+    options: DeadLetterOptions = { deadletterReason: "", deadLetterErrorDescription: "" }
```

Comment:
I wouldn't recommend using the default parameter syntax. I think when we compile it to ES6 our base target and ES5 for browsers, default parameter syntax is not transpiled correctly. Hence, it is better to avoid this syntax and set the default value inside the function if the parameter is null | undefined.

Reply:
Interesting! I wasn't aware of this limitation. 

All right then -- back to the first implementation :). Sorry for the distraction @ShivangiReja

Reply:
No problem, I will change this syntax.